### PR TITLE
Bug fix/app

### PR DESCRIPTION
**Pull request by hayleyw7**
# Description drawing

- Fix app name
- Fix how-to link to go to grid

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [ ] dev tools
- [ ] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "PoKedeX",
+  "name": "PoKedeX",
   "icons": [
```
Comment (Matt-Roden) [inline public/manifest.json:4]: Great!

File: src/components/App/App.js [hunk 2]
```diff
@@ -66,10 +66,18 @@ class App extends Component {
   };
 
   clearErrorMessage = () => {
-
     return this.setState({error: null})
   }
 
+  clearPokemon = (e) => {
```
Comment (Matt-Roden) [inline src/components/App/App.js:72]: Great work fixing this bug!

File: src/components/Help/Help.js [hunk 3]
```diff
@@ -18,6 +23,7 @@ const Help = () => {
               className='x-icon'
               src='Images/x-icon.png'
               align='right'
+              onClick={(e) => handleClick(e)}
```
Comment (Matt-Roden) [inline src/components/Help/Help.js:26]: smart

**Review by Matt-Roden — APPROVED** [review]

File: src/components/Help/Help.js [hunk 4]
```diff
@@ -4,7 +4,12 @@ import './Help.css';
 import { Link } from 'react-router-dom';
 
 
-const Help = () => {
+const Help = (props) => {
+
+  const handleClick = (e) => {
+    props.clearPokemon(e);
+  }
+
```
Comment (joanafbrito) [inline src/components/Help/Help.js:12]: 🤗🤗

**Review by joanafbrito — APPROVED** [review]
Awesome Group work!! Go team

**Pull request merged**